src/App.js

```diff
@@ -1,14 +1,132 @@
 import React, { Component } from 'react';
+import SearchBar from './Components/SearchBar';
+import MovieDetail from './Components/MovieDetail';
+import {Grid} from 'react-bootstrap';
+import MovieList from './Components/MovieList';
+// import MyMovieList from './Components/MyMovieList';
 import './App.css';
+import axios from 'axios';
+
 
 class App extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      searchText: '',
+      myMovies: [],
+      savedMovies: [],
+      movieDetails: {},
+      overview: '',
+      posterPath: '',
+      releaseDate: '',
+      title: '',
+
+    };
+  }
+  componentDidMount() {
+    axios.get('http://localhost:4000/movies')
+    .then(resp => {
+      this.setState({
+        myMovies: resp.data
+      });
+    });
+
+    // .catch(err => console.log(`Error! ${err}`));
+  }
+  handleSearchBarChange(event) {
+    this.setState({
+      searchText: event.target.value
+    });
+  }
+  handleSearchSubmit() {
+    const movie = this.state.searchText;
```

Comment:
Try to use descriptive variable names. Here I would expect this to be a full movie object. `movieName` would probably be clearer.